packages/shared/src/components/plus/GiftReceivedPlusModal.tsx

```diff
@@ -0,0 +1,125 @@
+import { cloudinaryGiftedPlusModalImage } from '../../lib/image';
+import { PlusList } from './PlusList';
+import { Button } from '../buttons/Button';
+import { getPlusGifterUser } from '../../graphql/users';
+import { generateQueryKey, RequestKey } from '../../lib/query';
+import { ProfileImageSize, ProfilePicture } from '../ProfilePicture';
+import Link from '../utilities/Link';
+
+interface GiftPlusModalProps extends ModalProps {
+  user?: UserShortProfile;
+}
+
+const GifterProfile = ({ gifter }: { gifter: UserShortProfile }) => (
+  <Link href={`/${gifter.username}`} passHref>
+    <a
+      className="flex items-center gap-2"
+      title={`View ${gifter.username}'s profile`}
+    >
+      <ProfilePicture user={gifter} size={ProfileImageSize.Medium} />
+      <Typography
+        bold
+        color={TypographyColor.Primary}
+        type={TypographyType.Callout}
+      >
+        {gifter.name}
+      </Typography>
+      <Typography
+        type={TypographyType.Footnote}
+        color={TypographyColor.Secondary}
+      >
+        @{gifter.username}
+      </Typography>
+    </a>
+  </Link>
+);
+
+const OpenSquadButton = () => (
+  <Button
+    className="w-full"
+    href="/squads/plus"
+    tag="a"
+    variant={ButtonVariant.Primary}
+  >{`See what's inside`}</Button>
+);
+
+export function GiftReceivedPlusModal({
+  user,
+  onRequestClose,
+  ...props
+}: GiftPlusModalProps): ReactElement {
+  const { data: gifter, isLoading } = useQuery({
+    queryKey: generateQueryKey(RequestKey.GifterUser),
+    queryFn: getPlusGifterUser,
+    enabled: Boolean(user.isPlus),
+  });
+
+  if (!gifter || isLoading) {
+    return null;
+  }
+
+  return (
+    <Modal
+      {...props}
+      drawerProps={{
+        displayCloseButton: false,
+      }}
+      isDrawerOnMobile
+      kind={Modal.Kind.FixedCenter}
+      onRequestClose={onRequestClose}
+      size={Modal.Size.Small}
+    >
+      <Modal.Body className="flex flex-1 tablet:!px-4">
+        <div className="flex flex-1 flex-col gap-4 overflow-y-auto tablet:overflow-auto">
+          <div className="flex flex-row justify-between ">
+            <PlusTitle type={TypographyType.Callout} bold />
+            <CloseButton
+              type="button"
+              size={ButtonSize.Small}
+              onClick={onRequestClose}
+            />
+          </div>
+          <GifterProfile gifter={gifter} />
+          <Typography bold type={TypographyType.Title1}>
+            Surprise! 🎁 {gifter.username} thought of you and gifted you a
+            one-year daily.dev Plus membership!
+          </Typography>
+          <img
+            src={cloudinaryGiftedPlusModalImage}
+            alt="gift pack with daily.dev logo"
+            height={140}
+            width={386}
+            className="h-auto w-full"
+          />
+          <PlusList className="overflow-clip !py-0" />
+        </div>
+        <div className="flex flex-col gap-4 tablet:hidden">
```

Comment:
I think we don't need this footer at all. We should just use the drawer's close button. The issue is, I don't think it should actually be called `Cancel`. It should rather be called `Close`.

Reply:
We have it in design, should we drop it? 
<img width="356" alt="image" src="https://github.com/user-attachments/assets/6f8f29cc-a2ce-4f46-bb63-e01968cf8a0f" />

Reply:
We should probably check with design/product team.

Reply:
Updated ✔️